applications/crossbar/priv/couchdb/schemas/system_config.speech.json

```diff
@@ -8,6 +8,15 @@
             "description": "speech asr api key",
             "type": "string"
         },
+        "asr_api_key_google": {
```

Comment:
These Google-specific keys should go under a "google" root key, or perhaps a "speech.google" system config document.